### ISSUE TITLE
[FR] Run Command

### ISSUE THREAD

**Issue by zcysxy** [twibiral/obsidian-execute-code#63]
Thanks for this amazing plugin!

A small feature request: add a command to run the codeblock when the cursor is in it. Then we can assign a hotkey to it, without the need to hit the `run` button using a mouse.

This is implemented in [Snippets Plugin](https://github.com/cristianvasquez/obsidian-snippets-plugin/blob/master/src/main.ts).

**Comment by mayurankv** [comment]
This would be brilliant! Can't believe how good this plugin is!!

**Comment by chlohal** [comment]
This would be a great feature to have, true! Are you referring to Live Preview mode specifically? I think that either source mode or preview mode would be a challenge -- I *think* we'd likely have to manage #129 first

**Comment by zcysxy** [comment]
@chlohal Yes, I only use Live Preview mode, and I think most users do the same. And I also think your pull request is a must-have feature before this feature request. Currently, this plugin replaces the codeblock with a new view, where the output shows. With your pull request, no custom view is needed, making a lot of features doable in Live Preview.
